pkgs/misc/vscode-extensions/default.nix

```diff
@@ -192,15 +277,20 @@ let
       };
 
       jnoortheen.nix-ide = buildVscodeMarketplaceExtension {
+        meta = with stdenv.lib; {
```

Comment:
```suggestion
        meta = with lib; {
```

Reply:
Thanks. Just added changed in batch. After PR is accepted, I'll submit "pkgs/misc/vscode-extensions/default.nix" normalized (alphabetical order, following same standard). Haven't done so yet because I don't want to mix with these new extensions. I was already working on this before "treewide: with stdenv.lib; in meta -> with lib;" change.